packages/wrangler/e2e/remote-binding/miniflare-remote-resources.test.ts

```diff
@@ -427,157 +497,266 @@ const testCases: TestCase<string>[] = [
 				"Failed to extract service ID from VPC service creation output"
 			);
 
-			onTestFinished(async () => {
+			helper.onTeardown(async () => {
 				await helper.run(`wrangler vpc service delete ${serviceId}`);
 			});
 
-			return serviceId;
-		},
-		remoteProxySessionConfig: (serviceId) => [
-			{
-				VPC_SERVICE: {
-					type: "vpc_service",
-					service_id: serviceId,
-				},
-			},
-		],
-		miniflareConfig: (connection, serviceId) => ({
-			vpcServices: {
-				VPC_SERVICE: {
-					service_id: serviceId,
-					remoteProxyConnectionString: connection,
+			return {
+				remoteProxySessionConfig: {
+					bindings: {
+						VPC_SERVICE: {
+							type: "vpc_service",
+							service_id: serviceId,
+						},
+					},
 				},
-			},
-		}),
-		matches: [
+				miniflareConfig: (connection) => ({
+					vpcServices: {
+						VPC_SERVICE: {
+							service_id: serviceId,
+							remoteProxyConnectionString: connection,
+						},
+					},
+				}),
+			};
+		},
+		expectFetchToMatch: [
 			// Since we're using a real tunnel but no actual network connectivity, Iris will report back an error
 			// but this is considered an effective test for wrangler and vpc service bindings
 			expect.stringMatching(/CONNECT failed: 503 Service Unavailable/),
 		],
-	}, */
+	},
 ];
 
-const mtlsTest: TestCase<{ certificateId: string; workerName: string }> = {
-	name: "mTLS",
-	scriptPath: "mtls.js",
-	setup: async (helper) => {
-		const certificateId = await helper.cert();
+if (!CLOUDFLARE_ACCOUNT_ID) {
+	describe.skip(
+		"Skipping remote bindings E2E tests because CLOUDFLARE_ACCOUNT_ID is not set"
+	);
+} else {
+	describe("Remote bindings (proxy enabled)", () => {
```

Comment:
Personally I find the wording `proxy enabled/disabled` quite confusing, I'd be more explicit here (since the describe text is anyways pretty short)

something like `(remote proxy session enabled/disabled)`